pkg/webui/components/link/index.js

```diff
@@ -220,24 +220,21 @@ const GlossaryLink = ({ glossaryId, term, hideTerm, primary, secondary, classNam
       tabIndex="-1"
       raw
```

Comment:
```suggestion
      external
```

The way we are using it now, it should also have the "external link" icon.